spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/File.scala

```diff
@@ -209,6 +212,18 @@ object File {
       if (null != fs) fs.close()
     }
   }
+  /**
+   * load  file according to url
+   * @param url
+   * @param dir
+   */
+  def downloadFromUrl(url: String, dir: String): Unit = {
```

Comment:
Enhance the original FileReader instead of providing a API like this. Or you need to modify everywhere which use model load

Reply:
I have enhanced FileReader by invoking this method in file.

Reply:
Then why add this method?

Reply:
As this method should only be used in this file, please make this method private.